src/modules/helpchan.ts

```diff
@@ -62,6 +77,20 @@ export class HelpChanModule extends Module {
 			} hours of inactivity or when you send !close.`,
 		);
 
+	OCCUPIED_EMBED_BASE = new MessageEmbed()
+		.setTitle('⌛ Occupied Help Channel')
+		.setColor(HOURGLASS_ORANGE);
+
+	occupiedEmbed(asker: User) {
+		return new MessageEmbed(this.OCCUPIED_EMBED_BASE)
+			.setDescription(occupiedMessage(asker.tag))
+			.setFooter(
+				`Closes after ${
+					dormantChannelTimeout / 60 / 60 / 1000
+				} hours of inactivity or when ${asker.username} sends !close.`,
```

Comment:
IMO this should be `when ${asker.toString()} sends`

Reply:
I was thinking it might be an excessive number of mentions, but honestly it'll probably help people who're skimming.

Reply:
AFAIK mentions in embeds don't do anything (e.g. `!lb` output)